### DISCUSSION TITLE
Why is `effect_wspeed` (effective wind speed for SPITFIRE) calculated at site level and not patch level?

### DISCUSSION THREAD

**Discussion by adrifoster** [NGEET/fates#1117] [General]
I noticed that the [`patch%effect_wspeed`](https://github.com/NGEET/fates/blob/1874511a2944d623f4b7a608cdacd9bc96cf8610/fire/SFMainMod.F90#L439C21-L439C34) is calculated using the total site-level average `tree_fraction`, `grass_fraction`, and `bare_fraction` 

It seems like this is being done as a comment says "to prevent extreme divergence"

Is this what we want? I suppose it makes sense that the wind would be influenced by all the patches and not just the one we're on. In that case should this not be a site-level variable?

**Comment by rosiealice** [comment]
Yeah, this is basically to prevent lock in of patches into their existing states. Some simulations I did forever ago had very strong influence of initial conditions wrt the impact of changing climate, and this was something I did to try and ameliorate this somewhat.

**Reply by adrifoster** [reply]
In that case is it okay if I make it a part of a site-level class?
